presqt/api_v1/serializers/resource.py

```diff
@@ -32,6 +32,3 @@ class ResourceSerializer(serializers.Serializer):
     size = serializers.IntegerField()
     hashes = HashSerializer()
     extra = serializers.DictField()
-
-
```

Comment:
Not sure if `HashSerializer` belongs in this module...

Reply:
OR GET RID OF IT :)